libs/openFrameworks/gl/ofGLUtils.cpp

```diff
@@ -764,6 +772,33 @@ string ofGLSLVersionFromGL(int major, int minor){
 #endif
 }
 
+#ifdef TARGET_OPENGLES
+int ofGLESVersionFromGL(){
+	const GLubyte * version = glGetString(GL_VERSION);
+	if(version != nullptr){
+		if(strstr((const char*)version, "ES 1.0")){
+			return 100;
+		}else if(strstr((const char*)version, "ES 1.1")){
+			return 101;
+		}else if(strstr((const char*)version, "ES-CM 1.1")){
+			return 101;
+		}else if(strstr((const char*)version, "ES 2.0")){
+			return 200;
+		}else if(strstr((const char*)version, "ES 3.0")){
```

Comment:
This should work wonders for real time checking. However my only worry is polling GL the best idea all the time. I think this function should still exist.... however should we rather do a current renderer check from outside of GL for real time / update loop ES version checks.